source/material_model/averaging.cc

```diff
@@ -345,9 +345,18 @@ namespace aspect
           average (averaging_operation,in.position,out.densities);
           average (averaging_operation,in.position,out.thermal_expansion_coefficients);
           average (averaging_operation,in.position,out.specific_heat);
+          average (averaging_operation,in.position,out.thermal_conductivities);
           average (averaging_operation,in.position,out.compressibilities);
           average (averaging_operation,in.position,out.entropy_derivative_pressure);
           average (averaging_operation,in.position,out.entropy_derivative_temperature);
+
+          std::vector<double> reaction_temp (out.reaction_terms.size());
+          for (unsigned int k=0; k<this->n_compositional_fields(); ++k)
+            {
+              average (averaging_operation,in.position,reaction_temp);
```

Comment:
hm. you don't set `reaction_temp` to anything here, so I think the `average` function has nothing to average other than zeros. No?